Components/GridPostsArchive/_style.scss

```diff
@@ -14,33 +13,76 @@
   }
 
   .posts-empty {
-    padding: 40px 0 0;
+    padding: 32px 0 0;
   }
 
   .post {
-    margin-bottom: $gutter-width;
-
-    @media (min-width: $breakpoint-tablet) {
-      margin-bottom: 0;
-    }
+    @include reset-list-item;
+    display: flex;
 
     &-link {
-      display: block;
-      text-decoration: none;
+      @include reset-link;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
     }
 
     .figure {
-      background-color: $color-text;
+      flex-shrink: 0; // IE11 bug fix (prevent image from stretchin')
```

Comment:
I think this can be removed as we're dropping IE11 support. Please also see for other IE11 support that can be removed.